students/WooseokJ/MailP2fixedWJFixed.py

```diff
@@ -0,0 +1,121 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+
+import sys
+import datetime
+import operator
+
+
+def menu():
+    print(''''Menu
```

Comment:
Because of how you're calling this function from _main()_, you should change this print to _return_, otherwise, _None_ will be printed each time you call _menu_. This happens because you're doing this:

`_input(menu())_ # input tries to print the output of menu, but menu() has no return statement, so it returns "None" by default.`